### PR TITLE
Change homepage background to yellow

### PR DESCRIPTION
**Pull request by jennyxing**
technical changes made:

original product request:

tests passed: yes
Visual QA passed: yes

### CODE REVIEW

**Comment by jennyxing** [comment]
please put in more details in the description

**Comment by AdamPie29** [comment]
Please put in even more details in the description.

**Comment by jennyxing** [comment]
from the designer: is a version of this live in a staging so I can take a look?

File: styles.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 body {
     font-family: 'Slackey', cursive;
-    background-color: blue;
+    background-color: yellow;
 }
 
 img {
```
Comment (jennyxing) [inline styles.css:8]: this is a comment attached to a line of code

File: styles.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 body {
     font-family: 'Slackey', cursive;
-    background-color: blue;
+    background-color: yellow;
```
Comment (jennyxing) [inline styles.css:5]: can you put color into a variable for use later on?

File: styles.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
 }
```
Comment (jennyxing) [inline styles.css:6]: make sure to indent propperly

**Review by jennyxing — COMMENTED** [review]
looks good to me. just a few minor style suggestions but feel free to merge and deploy.

File: styles.css (hunk 2 above)
Comment (Keegs12) [inline styles.css:5]: can you use the hex color value #f7e948 instead of "yellow".

**Review by Rebecca-Collins — APPROVED** [review]
Looks good!

**Review by Keegs12 — COMMENTED** [review]
Did I do it right?

File: styles.css (hunk 2 above)
Comment (jennyxing) [reply styles.css:5]: sure thing!

Comment (jennyxing) [inline styles.css:5]:
```suggestion
    background-color: #f7e948;
```

**Pull request merged**